docs/components/table.md

```diff
@@ -85,6 +85,77 @@ Lessen the table cell padding
   isSmall />
 ```
 
+### Sorting
+There are two props used to make the table sortable; ```sortOrder```, which is either 'ascending' or 'descending' and ```sortKey```, which tells the table which column is currently being sorted. If a sortKey exists, then clicking the
+table header will emit an Event called ```sortField``` which must be handled by the parent component to implement the
+actual sorting logic. A basic implementation of ```sortField``` is included in KTable and can be imported separately and used with a helper function in the parent.
+Once a table column with ```sortable``` is read, that column header will become clickable. An arrow then appears beside the table header, the state of the arrow depending on the sortOrder.
+In the following example, the sortField from KTable is being imported, and the table is able to be sorted by any of the three columns by clicking on the headers.
+
+<template>
+  <KTable
+    :options="tableOptions"
+    :sort-order="sortOrder"
+    :sort-key="sortKey"
+    @sort-field="sortFieldHelper"
+    />
+</template>
+
+```vue
+<template>
+<KTable
+    :options="tableOptions"
+    :sort-order="sortOrder"
+    :sort-key="sortKey"
+    @sort-field="sortFieldHelper"
+    />
+</template>
+<script>
+import { sortField } from '../../packages/KTable/KTable'
```

Comment:
for the example, we should use `'@kongponents/KTable'` to be closer to what a user of kongponents will use. Also, calling the helper "sortField" is a little confusing because the event name is also sortField. How about "defaultSorter"? Can we somehow reduce the boilerplate of a defaultSorter usage case? e.g.
```vue
<KTable
    :options="tableOptions"
    :sort-order="sortOrder"
    :sort-key="sortKey"
    @sort-field="key => defaultSorter(key, sortKey, sortOrder)"
    />
```

And then have defaultSorter actually mutate the value of sortOrder/sortKey directly without needing to do it in a `method`. How do you feel about renaming "sort-field" to just be `@sort`?

Reply:
yeah i tried this method out already it seems that mutating the value in the method doesn't really work and the changes do not actually persist, thats why I had to make the helper method to set the values onto `this`